hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -433,6 +439,43 @@ void setSnapshotSummary(Map<String, String> parameters, Snapshot currentSnapshot
     }
   }
 
+  private void setSchema(TableMetadata metadata, Map<String, String> parameters) {
+    parameters.remove(TableProperties.CURRENT_SCHEMA);
+    if (metadata.schema() != null) {
```

Comment:
Lets' have common method and re-use to set various fields like
```
private void setField(TableMetadata metadata, Map<String, String> parameters, String key, String value) {
   parameters.remove(key);
   if (value.length <= maxHiveTablepropertySize) {
      parameters.put(key, value);
   } else {
      LOG.warn("Not exposing {} in HMS since it exceeds {} characters", maxHiveTablePropertySize);
   }
}
```

Reply:
Made the change.